helpers/DeployHelper.ts

```diff
@@ -24,6 +25,9 @@ import { MCPaymentProxyModule } from "../ignition/modules/mcPayment";
 
 const SMT_MAX_DEPTH = 64;
 
+export type VerifierType = "mtpV2" | "sigV2" | "v3" | "authV2";
```

Comment:
It should be Groth16VerifierType but not VerifierType. Verifiers are different from Groth16Verifiers and confuse a lot